src/main/java/graphql/schema/idl/RuntimeWiring.java

```diff
@@ -192,11 +192,24 @@ private Builder() {
             ScalarInfo.GRAPHQL_SPECIFICATION_SCALARS.forEach(this::scalar);
         }
 
+        /**
+         * This sets strict mode as true or false. If strictMode is true, if things get defined twice, for example, it will throw a {@link StrictModeWiringException}.
+         *
+         * @return this builder
+         */
+        public Builder strictMode(boolean strictMode) {
+            this.strictMode = strictMode;
+            return this;
+        }
+
         /**
          * This puts the builder into strict mode, so if things get defined twice, for example, it will throw a {@link StrictModeWiringException}.
          *
          * @return this builder
+         *
+         * @deprecated strictMode default value changed to true, use {@link #strictMode(boolean)} instead
          */
+        @Deprecated(since = "2025-02-15", forRemoval = true)
```

Comment:
Setting `forRemoval = true` to give clear indication this is going to be removed soon